util/testenv/testenv.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+// Package testenv provides utility functions for tests.
```

Comment:
Mention that this package specifically does not depend on the testing package? Might even be worth locking that in with a test.

We have a package to do negative tests now! Search for `deptest.DepChecker` and look at existing ones.

Reply:
Done, PTAL